cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectorIT.java

```diff
@@ -67,24 +76,40 @@ public static void before() throws IOException {
                         .setEnabled(true)
         ));
 
-        cluster = new EmbeddedKafkaCluster(3);
-        cluster.start();
+        Map<String, String> kafkaClusterConfiguration = new HashMap<>();
+        kafkaClusterConfiguration.put("zookeeper.connect", "zookeeper:2181");
+
+        kafkaCluster = new StrimziKafkaCluster(kafkaClusterConfiguration);
+        kafkaCluster.start();
+
+        Properties properties = new Properties();
+        properties.setProperty(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, kafkaCluster.getBootstrapServers());
+        adminClient = AdminClient.create(properties);
     }
 
     @AfterAll
     public static void after() {
         vertx.close();
+        adminClient.close();
+        kafkaCluster.stop();
     }
 
     @BeforeEach
-    public void beforeEach() throws IOException, InterruptedException {
+    public void beforeEach() throws IOException, InterruptedException, ExecutionException {
         String connectClusterName = getClass().getSimpleName();
-        cluster.createTopics(connectClusterName + "-offsets", connectClusterName + "-config", connectClusterName + "-status");
+
+        CreateTopicsResult createTopicsResult = adminClient.createTopics(
+            Arrays.asList(
+                new NewTopic(connectClusterName + "-offsets", 3, (short) 3),
+                new NewTopic(connectClusterName + "-config", 3,  (short) 3),
+                new NewTopic(connectClusterName + "-status", 3,  (short) 3)
+            ));
+        createTopicsResult.all().get();
```

Comment:
Why is this needed? Shouldn't Connect create this automatically?

Reply:
I don't know I just re-write this code to be equivalent by instead of using Kafka cluster to create it I am using AdminClient. I can give it a try....and see what happens...

Reply:
I see that @tombentley was first, which wrote that `cluster.createTopics(getClass().getSimpleName() + "-offsets", getClass().getSimpleName() + "-config", getClass().getSimpleName() + "-status");`... I tried to remove that code and test also passed so I don't really know :D 🤷‍♂️